code/modules/busy_space/systems.dm

```diff
@@ -0,0 +1,602 @@
+	planets = list("Mandrake")
+	locations = list(
+		new /datum/lore/location("a vista on Mandrake, Oasis", list(ATC_LUX)),
+		new /datum/lore/location("a resort on Mandrake, Oasis", list(ATC_LUX, ATC_TRANS, ATC_FREIGHT, ATC_MED)),
+		new /datum/lore/location("Gilthari Casino in orbit around Mandrake, Oasis", list(ATC_LUX, ATC_TRANS)),
+		new /datum/lore/location("a Golden Crescent Alliance base in Oasis", list(ATC_TYPICAL)),
+		new /datum/lore/location("Colony Ardent, Oasis", list(ATC_TYPICAL, ATC_DIPLO)),
+		)
+
+/datum/lore/system/saint_columbia
+	name = "Saint Columbia"
+	desc = "Famous for constant attempted secessions, most recently as part of the Almach Crisis. Currently home to an enormous fleet base and a government that is totally not a junta."
+	locations = list( //transparently this is going to be pretty much identical to the Jahan's Post list with Almach-specific stuff afterwards
+		new /datum/lore/location("the Iserlohn platform in Saint Columbia", list(ATC_INDU, ATC_FREIGHT, ATC_TRANS)),
+		new /datum/lore/location("a dreadnought in Saint Columbia", list(ATC_MED, ATC_INDU, ATC_TRANS, ATC_FREIGHT)),
+		new /datum/lore/location("a battlecruiser in Saint Columbia", list(ATC_MED, ATC_INDU, ATC_TRANS, ATC_FREIGHT)),
+		new /datum/lore/location("a Fleet construction site in Saint Columbia", list(ATC_MED, ATC_INDU, ATC_TRANS, ATC_FREIGHT)),
+		new /datum/lore/location("a demonstration in Saint Columbia", list(ATC_DEF, ATC_MED)), // woof
+		new /datum/lore/location("Barrueco habitation dome, Saint Columbia", list(ATC_TYPICAL)), //ancient, ancient ic-news lore
+		new /datum/lore/location("a classified location in Saint Columbia", list(ATC_TYPICAL)),
+		new /datum/lore/location("a caravan in Saint Columbia", list(ATC_TYPICAL)),
+		new /datum/lore/location("Almach War wreckage in Saint Columbia", list(ATC_SALVAGE)),
+		)
+
+/datum/lore/system/raphael
+	name = "Raphael"
+	desc = "Positronic voter farms in the Crypt."
+	locations = list( //discord indicates they had some Troubles during the incursion but IDK anything fun to give them as a result
+		new /datum/lore/location("Yod habitation complex, Raphael", list(ATC_ALL_CIV)),
+		new /datum/lore/location("He habitation complex, Raphael", list(ATC_ALL_CIV)),
+		new /datum/lore/location("Waw habitation complex, Raphael", list(ATC_ALL_CIV)),
+		)
+
+/datum/lore/system/love
+	name = "Love"
+	desc = "Government is basically owned by a bunch of triads. Wellfare's really good though. Home to the ZMR."
+	planets = list("the planet") // ...ok. so. wiki indicates Love has a habitable world. there is no sense at all what it is called. this will generate locations in the form of "a dome on the planet in Love", which is bizzarely circumspect but basically fine. the alternative is giving them only space sites until someone names the planet.
```

Comment:
I think this is grammatically fine if the planet is also called Love, which is a _relatively_ common arrangement for minor colonies.